scripts/transforms/custom-value-css-box-shadow.transform.js

```diff
@@ -17,6 +19,17 @@ module.exports = {
       return 'none'
     }
 
+    if (isArray(value)) {
+      return value.map(val => {
+        const boxShadow = `${val.x}px ${val.y}px ${val.blur}px ${val.spread}px ${val.color}`
+        if (val.type === 'innerShadow') {
+          return `inset ${boxShadow}`
+        }
+
+        return boxShadow
+      })

```

Comment:
You shouldn't be returning array here. you could use `value.reduce()` or join array items with `value.map().join(', ')` to generate string.

Reply:
good point ;)